timemachine/fe/topology.py

```diff
@@ -319,6 +319,74 @@ def parameterize_nonbonded(self, ff_q_params, ff_lj_params):
         return combined_qlj_params, interpolated_potential
 
 
+class RelativeFreeEnergyForcefield(BaseTopology):
+    """
+    Used to run the same molecule under different forcefields.
+    Currently only changing the nonbonded parameters is supported.
+
+    The parameterize_* methods should be passed a list of parameters
+    corresponding to forcefield0 and forcefield1.
+    """
+
+    def __init__(self, mol, forcefield0, forcefield1):
+        """
+        Utility for working with a single ligand.
+
+        Parameter
+        ---------
+        mol: ROMol
```

Comment:
Nit: Annotations are sometimes `mol: Chem.Mol` and sometimes `mol: ROMol` -- is there a distinction?

Reply:
No, there isn't. I'll switch to Chem.Mol.